docs/guides/2-configuring-spree.md

```diff
@@ -26,6 +35,21 @@ namespace to Spree's default API routes, that respond as per the
 Follow the instructions on the Spree AMS Readme to install it into your Spree
 application.
 
+**TL;DR**
+
+
+```bash
+cd my_store_backend
+echo "\ngem 'spree_ams', :github => 'hhff/spree_ams', :branch => '3-0-alpha'" >> Gemfile
+bundle
+rails g spree:api:ams:install
+```
+
+In ```config/environments/development.rb``` add:
+```
+config.action_controller.asset_host = 'http://localhost:3000'
+```
+
```

Comment:
I'd like to remove these parts, as they're locked to specific versions (the rails version often matters here too), and may go out-of-date.  I think it's easy enough for a developer to consult the specific page on github, than for us to constantly keep in up-2-date